### DISCUSSION TITLE
i2c display support

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#226] [Announcements]
Thanks to @diyelectromusic there is now Initial support for I2C HD44780 displays.

I have updated the documentation accordingly.
Please test it, and don't forget to thank @diyelectromusic! :+1: 

 #225

**Comment by JoaquinSainz** [comment]
Great, where can i find the ic2 pinout guide?

**Reply by craigyjp** [reply]
Surely it's only 4 wires, SDA, SCK, +5v and Gnd. Its shown on the github page.

**Reply by craigyjp** [reply]
SCL, not SCK

**Comment by rocket-ron** [comment]
@probonopd I think you wrote previously that I2C only supports a single device - is that because of what Circle supports? So if using this display one wouldn’t be able to use the Audio DAC? Apologies if this is a silly question - trying to understand where the limitation is...

**Reply by probonopd** [reply]
I2S only supports one device. I2C supports multiple.

**Reply by Skeyelab** [reply]
> I2S only supports one device. I2C supports multiple.

So am I understanding correctly that I can use both an I2S DAC and a I2C display at the same time?  I'd wire the display to the same pins as the DAC, right?

**Comment by sou7611** [comment]
Yes, in fact many DACs (eg RASPIAUDIO) have I2C pins on them you have
utilize.  It's a bus connection, similar to CAN on cars.

**Comment by luisfcorreia** [comment]
I2S pins are to be used by DAC exclusively

I2C is a general purpose bus, where we want to connect our LCD Display but, in some DAC boards, it's also connected to the DAC chip

Chips in I2C have specific addresses so they can coexist on the same bus without issue
